python/graphscope/tests/unittest/test_app.py

```diff
@@ -234,8 +234,10 @@ def test_other_app_on_undirected_graph(
     assert np.allclose(r, triangles_result["undirected"])
 
     # louvain
-    ctx = louvain(p2p_project_undirected_graph, min_progress=50, progress_tries=2)
-    assert ctx is not None
+    ctx = louvain(p2p_project_undirected_graph, min_progress=0, progress_tries=2)
+    df = ctx.to_dataframe({"node": "v.id", "r": "r"})
+    community_num = len(df["r"].unique())
+    assert community_num == 103
```

Comment:
Is this number consistent over multiple runs?

Reply:
it should be consistent and still trying to figure out what make the result mismatch